framework/Samplers/AdaptiveDynamicEventTree.py

```diff
@@ -352,14 +346,15 @@ def _constructEndInfoFromBranch(self,model, myInput, info, cdfValues):
         self.inputInfo['SampledVarsPb'].update(precSample['SampledVarsPb'])
     self.inputInfo['PointProbability' ] = reduce(mul, self.inputInfo['SampledVarsPb'].values())*subGroup.get('conditionalPbr')
     self.inputInfo['ProbabilityWeight'] = self.inputInfo['PointProbability' ]
+    self.inputInfo.update({'ProbabilityWeight-'+key.strip():value for key,value in self.inputInfo['SampledVarsPb'].items()})
```

Comment:
Do you need to call the function ```_reassignSampledVarsPbToFullyCorrVars()``` in this sampler?

Reply:
it is done in the base class.

Reply:
right?

Reply:
I have checked, we have done this in both the base class and the inherited class, It seems we need to clean up.

Reply:
ok